include/zephyr/bluetooth/gatt.h

```diff
@@ -2005,6 +2008,8 @@ struct bt_gatt_read_params {
  *  The Response comes in callback @p params->func. The callback is run from
  *  the context specified by 'config BT_RECV_CONTEXT'.
  *  @p params must remain valid until start of callback.
+ *  If the received data length is invalid, the callback @p params->func will
+ *  called with the error @ref BT_ATT_ERR_INVALID_PDU.
```

Comment:
Ideally I'd only put this in one place. Having the text duplicated in 2 places makes it harder to maintain. I'd perhaps just keep it in bt_gatt_read_func_t